dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ForbiddenException.java

```diff
@@ -40,7 +40,7 @@
 @Getter
 @Accessors(chain = true)
 @OpenApi.Response(status = FORBIDDEN, value = WebMessageResponse.class)
-public final class ForbiddenException extends Exception implements Error {
+public final class ForbiddenException extends RuntimeException implements Error {
```

Comment:
Please revert to a checked exception and propagate that in the method signatures as `throws ForbiddenException`

Reply:
`ForbiddenException` kept as checked exception. We discussed this & the general approach will be to throw checked exceptions at the top level (controllers, services) and throw unchecked exception (`AccessDeniedException`) at lower levels (e.g. store).